devbox.lock

```diff
@@ -8,43 +8,15 @@
       "version": "1.6.27",
       "systems": {
         "aarch64-darwin": {
-          "outputs": [
```

Comment:
Please revert this file - unless you're making specific changes to it to e.g. add/update/remove tools this should not be part of this PR. I think this is changing here because you have a different version of devbox installed, but that shouldn't affect your usage of this file.

Reply:
reverted